README.md

```diff
@@ -69,7 +70,7 @@ environment.
 | `HOTJAR_ANALYTICS_TRACKING_ID`     | [Hotjar](https://www.hotjar.com/) tracking id                                                                                                              |                           | No         |
 | `BASIC_AUTH`                       | An MD5 encrypted [htpasswd](https://httpd.apache.org/docs/2.4/misc/password_encryptions.html)                                                              | test:test                 |            |
 | `DISABLE_GOOGLE_SEARCH`            | Set to 'true' to disable the metadata which enables [Google Sitelinks searchbox](https://developers.google.com/search/docs/data-types/sitelinks-searchbox) | false                     |            |
-| `DATE`                             | Set the date used to determine if a bank holiday message should be displayed                                                                               |                           | No         |
+| `DATETIME`                         | Set the datetime used to determine if a bank holiday message should be displayed or the behaviour of the results page                                                                  |                           | No         |
```

Comment:
The description isn't entirely clear upon first reading, but I'm struggling to think of a better way to word it. 
There's more spaces on the line than other env vars so the table fields don't line up.

Reply:
I've given it a go i.e. trying to clarify the use of the env var.